### DISCUSSION TITLE
Update EasyDutch contentURL and cdnURLs

### DISCUSSION THREAD

**Discussion by Nomes77** [uBlockOrigin/uAssets#17919] [General] [answered]
After I read
> The idea is to not get into trouble with bandwidth, not wait for
bandwidth issues to arise before doing something about it.
> 
> uBO's user base keeps growing, and this of course directly affect
bandwidth usage, which is offered freely by various entities.
Let's not abuse this free resource.

from https://github.com/uBlockOrigin/uAssets/commit/61b00335d0b9bb859e50bb583e9d6c7646281121. I, as [EasyDutch](https://github.com/EasyDutch-uBO/EasyDutch/) owner, did some changes.

I, therefore, propose `https://raw.githubusercontent.com/EasyDutch-uBO/EasyDutch/gh-pages/EasyDutch.all.txt` as the new `contentURL`.
And `https://easydutch-ubo.github.io/EasyDutchCDN/EasyDutch.all.txt`, `https://cdn.statically.io/gh/EasyDutch-uBO/EasyDutch/gh-pages/EasyDutch.all.txt`, `https://cdn.jsdelivr.net/gh/EasyDutch-uBO/EasyDutchCDN@main/EasyDutch.all.txt` as new `cdnURLs` (https://github.com/gorhill/uBlock/blob/master/assets/assets.json#L576-L589)

**Comment by gorhill** [comment]
https://github.com/gorhill/uBlock/commit/42106cf5ff390304e0bebc63f03a7c109bd84106

**Reply by gorhill** [reply]
Here is your stats for jsDelivr: <https://www.jsdelivr.com/package/gh/EasyDutch-uBO/EasyDutch?tab=stats>, so overall network traffic should be 3x time of what jsDelivr report since it's one of three CDNs.

**Reply by Nomes77** [reply]
@gorhill Thank you for your quick reponse.
Only one comment, you made the `main` branch the `contentURL`. However it should be the `gh-pages`, since I add timestamps to it.
So the url is `https://raw.githubusercontent.com/EasyDutch-uBO/EasyDutch/gh-pages/EasyDutch.txt`

**Comment by Nomes77** [comment, marked as answer]
Answered with https://github.com/gorhill/uBlock/commit/5e46978b8cd511f5b8364e811da8942e68d36acb
